dfastmi/batch.py

```diff
@@ -2417,11 +2424,11 @@ def detect_areas(dzgemi, dzmin, EFCi, wght_area_tot, areai, wbin, wthresh, sifac
         dzgemi_filtered[sub_areai != ia] = 0.0
         sub_area_list.append(sub_areai == ia)
         
-        #dfastmi.io.log_text("sed_vol",dict = {"ia": ia+1, "nr": 1})
+        #ApplicationSettingsHelper.log_text("sed_vol",dict = {"ia": ia+1, "nr": 1})
```

Comment:
Should be removed in batch refactoring, as is the comment line below.

Reply:
I agree, let's do this in batch refactoring